starboard/nplb/media_buffer_test.cc

```diff
@@ -48,13 +51,52 @@ constexpr SbMediaVideoCodec kVideoCodecs[] = {
 };
 
 constexpr SbMediaType kMediaTypes[] = {
-    kSbMediaTypeAudio, kSbMediaTypeVideo,
+    kSbMediaTypeAudio,
+    kSbMediaTypeVideo,
 };
 
-// Minimum audio and video budgets required by the 2020 Youtube Software
+// Minimum audio and video budgets required by the 2024 Youtube Hardware
 // Requirements.
 constexpr int kMinAudioBudget = 5 * 1024 * 1024;
-constexpr int kMinVideoBudget = 30 * 1024 * 1024;
+constexpr int kMinVideoBudget1080p = 30 * 1024 * 1024;
+constexpr int kMinVideoBudget4kSdr = 50 * 1024 * 1024;
+constexpr int kMinVideoBudget4kHdr = 80 * 1024 * 1024;
+constexpr int kMinVideoBudget8k = 200 * 1024 * 1024;
+
+int GetVideoBufferBudget(SbMediaVideoCodec codec,
+                         int frame_width,
+                         bool is_hdr) {
+  if (codec != kSbMediaVideoCodecAv1 && codec != kSbMediaVideoCodecH264 &&
+      codec != kSbMediaVideoCodecVp9) {
+    return kMinVideoBudget1080p;
+  }
+
+  static const bool ksupports8k =
```

Comment:
Nit: Shouldn't these also be camelcase~ish, like `kSupports8k` ?

Reply:
Whoops, you're right. Fixed